packages/@uppy/core/src/Uppy.js

```diff
@@ -216,7 +216,7 @@ class Uppy {
 
     if (newOpts.locale) {
       this.iteratePlugins((plugin) => {
-        plugin.setOptions()
+        plugin.setOptions({})
```

Comment:
Wouldn't that be a more logical fix? I don't see how passing an empty object would be better than nothing at all.
```suggestion
        plugin.setOptions(newOpts)
```

Reply:
i update code.